app/src/main/java/com/mealsmadeeasy/model/UserProfile.kt

```diff
@@ -1,8 +1,26 @@
 package com.mealsmadeeasy.model
 
+const val FEET_TO_METERS = 0.3048
+const val INCHES_TO_METERS = 0.0254
+const val POUNDS_TO_KILOGGRAMS = 0.453592
+
 data class UserProfile(
         val gender: Gender,
         val birthday: Timestamp,
         val height: Inches,
         val weight: Pounds
-)
+) {
+    companion object {
+        fun calculateBMI(feet: Int, inches: Int, pounds: Int): Double {
```

Comment:
This can totally be a val on the UserProfile. You can do stuff like this:
```
data class UserProfile(...) {
    val bmi: Double
        get() = /* math */
}
```